lms/services/organization.py

```diff
@@ -135,14 +135,17 @@ def create_organization(self, name=None) -> Organization:
 
         return org
 
-    def update_organization(self, organization, name=None, enabled=None):
+    def update_organization(self, organization, name=None, enabled=None, notes=None):
```

Comment:
`notes` could become a `settings` parameter here or similar, we only have `notes` right now so that can wait.